foundry.toml

```diff
@@ -4,9 +4,4 @@ out = 'out'
 libs = ['lib']
 fs_permissions = [{ access = "read-write", path = "./"}]
```

Comment:
it looks like this permission can be more strict with:
```toml
fs_permissions = [
    { access = "read", path = "./script/input/1/integration.json"},
    { access = "read", path = "./out/ArbitrumDomain.sol/ArbSysOverride.json"}
]
```
that way we don't give read-write permission to the whole repo

Reply:
Is there any danger with reading/writing from the whole repo? I'm struggling to come up with a case where sensitive info would be present.

Reply:
Done.

Reply:
> Is there any danger with reading/writing from the whole repo? I'm struggling to come up with a case where sensitive info would be present.
> 
> Perhaps as a compromise I'll set it to read-only in `script/input` and the specific ArbSysOverride just so we don't have to add these everytime a new config is added.

I think that works.  I vaguely recall seeing something about forge/foundry having a vuln related to the fs access.  I couldn't find it yesterday, but in general I think we should follow the principle of least privilege especially since this will be imported to other repos that would then inherit the permissions.

Thanks for updating

Reply:
Yeah it's a fair point.

Reply:
Not sure if it's possible here but I'd be interested to see if something with r/w access could write out a symlink and break out of the path.

Reply:
Can give it a try. This seems like a huge incompetence of the Rust team if this is possible though.